internal/operators/lvm/manifest.go

```diff
@@ -32,11 +35,33 @@ func Manifests() (map[string][]byte, []byte, error) {
 	return openshiftManifests, lvmcluster, nil
 }
 
-func getSubscription() ([]byte, error) {
+func getSubscription(cluster *common.Cluster) ([]byte, error) {
+
+	ocpVersion, err := version.NewVersion(cluster.OpenshiftVersion)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	minOCPVersionForLVM, err := version.NewVersion(lvmsMinOpenshiftVersion)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	if ocpVersion.LessThan(minOCPVersionForLVM) {
```

Comment:
```suggestion
        operatorSubscriptionName := LVMS_SUBSCRIPTION_NAME
        operatorSubscriptionSpecName := LVMS_SUBSCRIPTION_SPEC_NAME
	if ocpVersion.LessThan(minOCPVersionForLVMS) {
	         operatorSubscriptionName := LVMO_SUBSCRIPTION_NAME
                 operatorSubscriptionSpecName := LVMO_SUBSCRIPTION_SPEC_NAME
                 }
                 data := map[string]string{
			"OPERATOR_NAMESPACE":              Operator.Namespace,
			"OPERATOR_SUBSCRIPTION_NAME":     operatorSubscriptionName,
                        "OPERATOR_SUBSCRIPTION_SPEC_NAME": operatorSubscriptionSpecName,
		}
		return executeTemplate(data, "LvmSubscription", LvmSubscription)
	}
```

Reply:
declared const in one place, config file